examples/jsm/nodes/display/PassNode.js

```diff
@@ -91,14 +92,29 @@ class PassNode extends TempNode {
 
 	}
 
+	getViewZNode() {
```

Comment:
Maybe we can do these just as getters? I.e. something like `get viewZ()`, `get depth()` instead of `getViewZNode()`, `getDepthNode()`?

Reply:
I think it's a good idea move to getters, I just will add `*Node` suffix.